src/views/blogPosts/MVP.vue

```diff
@@ -351,13 +318,7 @@
       MVP reliable and usable.
     </p>
     <p>
-      <img
-        :src="img('mvp-pyramid-640.png')"
-        :srcset="`
-          ${img('mvp-pyramid-640.png')} 1x,
-          ${img('mvp-pyramid-1000.png')} 2x
-        `"
-      />
+      <img class="img-fluid w-100" :src="img('mvp-pyramid.png')" />
```

Comment:
in general, it would not be a bad idea, to have multiple versions of graphics, to optimize for speed and performance and only use an image with better resolution if an image could really be displayed in such resolution. But for know, because those graphics are not prepared anyway, this in fact can be removed.